src/components/landingPage/articleCard/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './articleCard.scss';
+import UserIcon from './userIcon';
+import RatingArticleCardComponent from './ratingComponent';
+import ArticleCardTextComponent from './articleCardTextComponent';
+
+const optionImg = 'https://images.pexels.com/photos/1193743/pexels-photo-1193743.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260';
+
+const ArticleCard = ({
```

Comment:
The articleCard should be among the common components to make it easy to reuse

Reply:
I worked on the feedback